extensions/amp-youtube/1.0/youtube.js

```diff
@@ -138,33 +141,25 @@ export function Youtube({
       );
     }
     if (data.event == 'infoDelivery' && playerState != undefined) {
-      dispatchCustomEvent(currentTarget, PlayerStates[playerState.toString()]);
+      dispatchVideoEvent(currentTarget, PlayerStates[playerState.toString()]);
     }
     if (data.event == 'infoDelivery' && info['muted']) {
-      dispatchCustomEvent(currentTarget, mutedOrUnmutedEvent(info['muted']));
+      dispatchVideoEvent(currentTarget, mutedOrUnmutedEvent(info['muted']));
       return;
     }
   };
 
-  const makeMethodMessage = (method) => {
-    return JSON.stringify(
-      dict({
-        'event': 'command',
-        'func': methods[method],
-      })
-    );
-  };
-
   return (
-    <VideoIframe
+    <VideoWrapper
+      {...rest}
```

Comment:
I'm not convinced all of the below props should overwrite the given ones.

For instance:
- Is there any situation where we would want `component` to be user overridable? 
- We probably also want `Youtube` to take an `onLoad` and `onMessage` from the user props that we also call with our internal `onIframeLoad` and `onMessage`, wdyt?

Reply:
No, I think all of these are strictly props that "YouTube" must provide for a `VideoIframe` and shouldn't be customizable.

> Is there any situation where we would want component to be user overridable?

Shouldn't be customizable - this is just an internal implementation detail of the youtube player that it works via a video iframe.

> We probably also want Youtube to take an onLoad and onMessage from the user props that we also call with our internal onIframeLoad and onMessage, wdyt?

We do take `onLoad`. It's actually important. It's not directly handled by the `YouTube`, but it's propagated to the `VideoWrapper` via `{...rest}` and that should do the correct handling. The `onIframeLoad` is specific to the `VideoIframe`. This is an important part because `onLoad` sounds very generic, but it potentially means different things for an iframe vs a video player.

I don't see why a user would supply `onMessage` here, however - it'd seem like too much of an internal detail?

Reply:
Sounds good, thanks for clarifying these points.